kube/kd/deployment.yaml

```diff
@@ -0,0 +1,199 @@
+                  name: {{.KUBE_NAMESPACE}}-audit-sqs
+                  key: secret_access_key
+            - name: DOCUMENT_QUEUE
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-document-sqs
+                  key: sqs_queue_url
+            - name: DOCUMENT_QUEUE_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-document-sqs
+                  key: access_key_id
+            - name: DOCUMENT_QUEUE_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-document-sqs
+                  key: secret_access_key
+            - name: DOCUMENT_DLQ_QUEUE
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-document-sqs
+                  key: sqs_dlq_url
+            - name: DOCUMENT_DLQ_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-document-sqs
+                  key: access_key_id
+            - name: DOCUMENT_DLQ_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-document-sqs
+                  key: secret_access_key
+            - name: NOTIFY_QUEUE
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-notify-sqs
+                  key: sqs_queue_url
+            - name: NOTIFY_QUEUE_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-notify-sqs
+                  key: access_key_id
+            - name: NOTIFY_QUEUE_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-notify-sqs
+                  key: secret_access_key
+            - name: NOTIFY_DLQ_QUEUE
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-notify-sqs
+                  key: sqs_dlq_url
+            - name: NOTIFY_DLQ_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-notify-sqs
+                  key: access_key_id
+            - name: NOTIFY_DLQ_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{.KUBE_NAMESPACE}}-notify-sqs
+                  key: secret_access_key
+          resources:
+            limits:
+              cpu: 200m
+              memory: 512Mi
+            requests:
+              cpu: 200m
+              memory: 512Mi
+          ports:
+            - name: http
+              containerPort: 8080
+          startupProbe:
+            httpGet:
+              path: /actuator/health/liveness
+              port: http
+              httpHeaders:
+                - name: X-probe
+                  value: kubelet
+            initialDelaySeconds: 10
```

Comment:
These are high due to long startup time - something to look at going forward but think its the aws library initialisation.